### PR TITLE
Create pvc docs

### PR DESCRIPTION
**Pull request by evanluc**
### Fixes [MC-12122](https://cloud-ops.atlassian.net/browse/MC-12122)

Docs for create pvc. Tested in my browser for both standalone and extended plugins.

### CODE REVIEW

File: source/includes/kubernetes/_k8_persistentvolumeclaims.md [hunk 1]
```diff
@@ -172,3 +172,67 @@ Retrieve a persistent volume claim and all its info in a given [environment](#ad
...
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/persistentvolumeclaims</code>
+
+Create a persistent volume claim in a given [environment](#administration-environments).
+
+| Required Attributes                             | &nbsp;                                                                                                                                                                                                          |
+| ----------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `apiVersion` <br/>_string_                      | The api version (versioned schema) of the persistent volume claim                                                                                                                                               |
```
Comment (lamminade) [inline source/includes/kubernetes/_k8_persistentvolumeclaims.md]: can we throw periods at the ends of these lines?

File: source/includes/kubernetes_extension/_k8_persistentvolumeclaims.md [hunk 2]
```diff
@@ -180,3 +180,70 @@ Retrieve a persistent volume claim and all its info in a given [environment](#ad
...
+
+| Required                   | &nbsp;                                                               |
+| -------------------------- | -------------------------------------------------------------------- |
+| `cluster_id` <br/>_string_ | The id of the cluster in which to list the persistent volume claims. |
+
+| Required Attributes                             | &nbsp;                                                                                                                                                                                                          |
+| ----------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `apiVersion` <br/>_string_                      | The api version (versioned schema) of the persistent volume claim                                                                                                                                                                   |
```
Comment (lamminade) [inline source/includes/kubernetes_extension/_k8_persistentvolumeclaims.md]: same comment for here

**Review by lamminade — APPROVED** [review]

**Review by GuillaumeVialle — APPROVED** [review]
I think you could add dots at the end of the attributes description for consistency.

File: source/includes/kubernetes/_k8_persistentvolumeclaims.md [hunk 3]
```diff
@@ -172,3 +172,67 @@ Retrieve a persistent volume claim and all its info in a given [environment](#ad
...
+| `metadata` <br/>_object_                        | The metadata of the persistent volume claim                                                                                                                                                                     |
+| `metadata.name` <br/>_string_                   | The name of the persistent volume claim                                                                                                                                                                         |
+| `spec` <br/>_object_                            | The spec for the persistent volume claim                                                                                                                                                                        |
+| `spec.accessModes` <br/>_array_                 | A list of access modes, the options are: ReadWriteOnce, ReadOnlyMany and ReadWriteMany                                                                                                                          |
```
Comment (evanluc) [inline source/includes/kubernetes/_k8_persistentvolumeclaims.md]:
```suggestion
| `apiVersion` <br/>_string_                      | The api version (versioned schema) of the persistent volume claim.                                                                                                                                               |
| `metadata` <br/>_object_                        | The metadata of the persistent volume claim.                                                                                                                                                                     |
| `metadata.name` <br/>_string_                   | The name of the persistent volume claim.                                                                                                                                                                         |
| `spec` <br/>_object_                            | The spec for the persistent volume claim.                                                                                                                                                                        |
| `spec.accessModes` <br/>_array_                 | A list of access modes, the options are: ReadWriteOnce, ReadOnlyMany and ReadWriteMany.                                                                                                                          |
```

File: source/includes/kubernetes_extension/_k8_persistentvolumeclaims.md [hunk 4]
```diff
@@ -180,3 +180,70 @@ Retrieve a persistent volume claim and all its info in a given [environment](#ad
...
+| `metadata` <br/>_object_                        | The metadata of the persistent volume claim                                                                                                                                                                                         |
+| `metadata.name` <br/>_string_                   | The name of the persistent volume claim                                                                                                                                                                                             |
+| `spec` <br/>_object_                            | The spec for the persistent volume claim                                                                                                                                                                        |
+| `spec.accessModes` <br/>_array_                 | A list of access modes, the options are: ReadWriteOnce, ReadOnlyMany and ReadWriteMany                                                                                                                          |
```
Comment (evanluc) [inline source/includes/kubernetes_extension/_k8_persistentvolumeclaims.md]:
```suggestion
| ----------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `apiVersion` <br/>_string_                      | The api version (versioned schema) of the persistent volume claim.                                                                                                                                                                   |
| `metadata` <br/>_object_                        | The metadata of the persistent volume claim.                                                                                                                                                                                         |
| `metadata.name` <br/>_string_                   | The name of the persistent volume claim.                                                                                                                                                                                             |
| `spec` <br/>_object_                            | The spec for the persistent volume claim.                                                                                                                                                                        |
| `spec.accessModes` <br/>_array_                 | A list of access modes, the options are: ReadWriteOnce, ReadOnlyMany and ReadWriteMany.                                                                                                                          |
```

**Pull request merged**